helpers/uploader.rb

```diff
@@ -0,0 +1,13 @@
+module Uploader
+  FILE_PATH = 'accounts.yml'.freeze
+
+  def load_data
+    return [] unless File.exist?(FILE_PATH)
+
+    YAML.load_file(FILE_PATH)
+  end
+
+  def save_data(data)
+    File.open(FILE_PATH, 'w') { |f| f.write data.to_yaml }
```

Comment:
What happens if you save data, and the file doesn't exist?

Reply:
Method ```open``` with parameter ```w``` will create a new file and write data there